data/json/professions.json

```
@@ -537,6 +537,22 @@
       "female": [ "bra", "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "vid_blogger",
```

```suggestion
    "id": "video_blogger",
```

No reason to shorten that.